src/coreclr/src/tools/crossgen2/ILCompiler.ReadyToRun/ObjectWriter/R2RPEBuilder.cs

```diff
@@ -350,8 +366,16 @@ public void AddSections(MapFileBuilder mapFileBuilder)
         /// we're performing the same transformation on Windows where it is a no-op.
         /// </summary>
         /// <param name="outputStream"></param>
-        private void UpdateSectionRVAs(Stream outputStream)
+        private void UpdateSectionRVAs(Stream outputStream, int? updateSectionAlign)
         {
+            if (updateSectionAlign.HasValue)
```

Comment:
We already have similar PE header patch-up functions, `ApplyMachineOSOverride` and `SetPEHeaderTimeStamp` in this file. Could you follow their pattern?